src/DependencyInjection/Configuration.php

```diff
@@ -18,8 +18,13 @@ class Configuration implements ConfigurationInterface
      */
     public function getConfigTreeBuilder()
     {
-        $treeBuilder = new TreeBuilder();
-        $rootNode = $treeBuilder->root('m6_statsd');
+        if (method_exists(new TreeBuilder(), 'getRootNode')) {
```

Comment:
```suggestion
```

it's ok if we drop support for older versions

if you could just add this line to the `require` key in composer.json:

 ```json
"symfony/config": "^4.2"
```